src/7-to-8/summary.rst

```diff
@@ -166,6 +167,13 @@ other hosts on the same platform to interact with task jobs.
           # Platforms can have the same hosts with different job runners.
           platform = hpc1_background
 
+
+.. warning::
+
+   Cylc 8 will attempt to pick a sensible platform for your Cylc 7 settings,
+   but this upgrade logic will be removed at Cylc 9.
```

Comment:
```suggestion
   These deprecated settings will be removed at Cylc 9.
``` 
(The upgrade logic gets removed, but perhaps more importantly the deprecated logic becomes illegal/obsolete.